docs/ksql-clickstream-demo/non-docker-clickstream.rst

```diff
@@ -0,0 +1,364 @@
+.. _ksql_clickstream_non_docker:
+
+Clickstream Analysis with KSQL running locally
+==============================================
+
+These steps will guide you through how to setup your environment and run
+the clickstream analysis demo.
+
+Prerequisites
+-------------
+
+-  :ref:`Confluent 3.3.0 <installation>` locally
+
+   -  Make sure you only have one broker running on the host
+
+-  `ElasticSearch <https://www.elastic.co/guide/en/elasticsearch/guide/current/running-elasticsearch.html>`__
+-  `Grafana <http://docs.grafana.org/installation/>`__
+-  `Git <https://git-scm.com/downloads>`__
+-  `Maven <https://maven.apache.org/install.html>`__
+-  Java: Minimum version 1.8. Install Oracle Java JRE or JDK >= 1.8 on
+   your local machine
+
+1.  Clone the Confluent KSQL repository.
+
+    .. code:: bash
+
+        $ git clone git@github.com:confluentinc/ksql.git
+
+2.  Change directory to the ``ksql`` directory and compile the KSQL
+    code.
+
+    .. code:: bash
+
+        $ cd ksql
+        $ mvn clean compile install -DskipTests
+
+3.  Copy the Kafka Connect Elasticsearch configuration file
+    (``ksql/ksql-clickstream-demo/demo/connect-config/null-filter-4.0.0-SNAPSHOT.jar``)
+    to your Confluent installation ``share`` directory
+    (``confluent-3.3.0/share/java/kafka-connect-elasticsearch/``).
+
+    .. code:: bash
+
+        cp ksql-clickstream-demo/demo/connect-config/null-filter-4.0.0-SNAPSHOT.jar <path-to-confluent-3.3.0>/share/java/kafka-connect-elasticsearch/
+
+4.  From your terminal, start the Confluent Platform. It should be
+    running on default port 8083.
+
+    .. code:: bash
+
+        $ <path-to-confluent-3.3.0>/bin/confluent start
```

Comment:
3.3.0?

Reply:
Also, perhaps we should rename `<path-to-confluent-3.3.0>` so that we don't need to update these references every time we make a release?  For example, we say `<path-to-ksql>` below.

Reply:
good point - i changed to `path-to-confluent`